spec/logging.md

```diff
@@ -22,6 +22,7 @@ to the console.
 - `--quiet` or `-q` is an alias to `--log-level=error`
 - `--log-file=<path>` - specified a file to write to, logs will be written in JSON format to the file
   (but a human-readable format to the console)
+- `--stack-trace` - prints stack trace along with error messages
```

Comment:
I did `--stack-trace` instead of `--trace` because I saw it a few times already, however, I'm not sure which one is better. WYT?

Reply:
Either one is fine. It is to be used infrequently by devs, it doesn't have to be short. `--stack-trace` is probably better